scene/gui/rich_text_label.cpp

```diff
@@ -1435,7 +1435,7 @@ void RichTextLabel::_notification(int p_what) {
 			while (ofs.y < size.height && from_line < main->lines.size()) {
 				visible_paragraph_count++;
 				visible_line_count += _draw_line(main, from_line, ofs, text_rect.size.x, base_color, outline_size, outline_color, font_shadow_color, use_outline, shadow_ofs);
-				ofs.y += main->lines[from_line].text_buf->get_size().y;
+				ofs.y += main->lines[from_line].text_buf->get_size().y + get_theme_constant(SNAME("line_separation"));
```

Comment:
The same change should be done in the `RichTextLabel::_find_click`, (on line 1130) or clicks and selection won't take line separation into account.

Reply:
If your okay with that, i can open a issue and pr.

Reply:
You don't need to open issue, just make a PR directly and link to this comment.

Reply:
https://github.com/godotengine/godot/pull/51933